docs/coding-style.md

```diff
@@ -45,23 +45,38 @@ Variable name `sut` (System Under Test) is recommended in unit tests that really
 * Dot before an invocation `.Method()` is placed at the beginning of a line.
 * The comma separating arguments is placed at the end of a line.
 * Method declaration parameters should be on the same line. If S103 is violated, parameters should be placed each on a separate line; the first parameter should be on the same line with the declaration; the other parameters should be aligned with the first parameter.
-```
-public void MethodWithManyParameters(int firstParameter,
-                                     string secondParameter,
-                                     Function<int, string, string> complexParameter);
-```
+    ```
+    public void MethodWithManyParameters(int firstParameter,
+                                         string secondParameter,
+                                         Function<int, string, string> complexParameter);
+    ```
+* Chained invocations and member accesses violating S103 can have a chain on the first line until the first lambda, every other `.Invocation()` or `.Member` should be on a separate line, aligned with a left-most single indendation.
+    ```
+    object.Property.Method().Select(x => x.Something)
+        .Where(x => x != null)
+        .OrderBy(x => x.Rank)
+        .ToArray();
+    ```
+* Long ternary operator statements should have `?` and `:` on separate lines, aligned with a left-most single indendation.
+    ```
+    object.Property is SomeType something
+    && something.AnotherProperty is OtherType other
+    && other.Value == 42
+        ? object.Parent.Value
+        : object;
+    ```
 * Method invocation arguments should be placed on the same line only when they are few and simple. Otherwise, they should be placed on separate lines. The first argument should be on a separate line, aligned with a left-most single indendation.
-```
-object.MethodName(
-    firstArgument,
-    x => x.Bar(),
-    thirdArgument.Property);
-```
+    ```
```

Comment:
This +4 space fencing is needed for github to properly align the code block under the bullet point and to fix the nested bullet point for `Exception from this rule:` below.

As seen [here](https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md#multi-line-statements), the nested bullet point is not understood by GitHub in the previous version